django_lightweight_queue/utils.py

```diff
@@ -124,6 +128,35 @@ def load_all_tasks():
     import_all_submodules('tasks', app_settings.IGNORE_APPS)
 
 
+def block_for_time(
+    should_continue_blocking: Callable[[], bool],
+    timeout: datetime.timedelta,
+    check_frequency: datetime.timedelta = THIRTY_SECONDS,
+) -> bool:
+    """
+    Block until a cancellation function or timeout indicates otherwise.
+
+    Returns whether or not the timeout was encountered.
+    """
+    if not should_continue_blocking:
```

Comment:
Should this be `should_continue_blocking()`? I think the callable is always going to be truthy (ignoring weird `__bool__` implementations).

(Same for below)

Reply:
:facepalm: 774cd39